internal/guest/gcserr/errors.go

```diff
@@ -45,15 +45,17 @@ const (
 	// HrVmcomputeInvalidState is the HRESULT for:
 	//
 	// The requested virtual machine or container operation is not valid in the
-	// current state
+	// current state.
 	HrVmcomputeInvalidState = Hresult(-2143878907) // 0x80370105
 	// HrVmcomputeSystemAlreadyStopped is the HRESULT for:
 	//
 	// The virtual machine or container with the specified identifier is not
-	// running
+	// running.
 	HrVmcomputeSystemAlreadyStopped = Hresult(-2143878896) // 0x80370110
 )
 
+// TODO: update implementation to use go1.13 style errors with `errors.As` and co.
+
```

Comment:
was this one of the exceptions?

Reply:
it heavily relies on `"github.com/pkg/errors"` implementation details, so I wanted to leave the change for a future PR, since itll likely be fairly involved